### ISSUE TITLE
Updated installation instructions

### ISSUE THREAD

**Issue by dekhaus** [nodes-vapor/admin-panel#121]
Hi All

I'm new to Swift/Vapor and I'm trying to configure a new Vapor3 'web' app to use AdminPanel (the vapor-3 branch).  The instructions confuse me.  Some questions ...

- its not clear from the instructions how to reference the vapor-3 branch
- the instructions say to create a few json config files - but doesn't say what their locations are
- I suspect (but am not sure) that the instructions in the 'master' branch are Vapor2 specific.  That doesn't help if you're new to the eco-system and want to use Vapor3

Can Vapor3 instructions be created ?

Thanks
Dave

**Comment by joscdk** [comment]
Hi,

For referencing branch, you should be able to do something like:

```swift
.package(url: "https://github.com/nodes-vapor/admin-panel.git", .branch("vapor-3"))
```

For json files, these are for Vapor 2, which used json config files, however this is no longer valid for Vapor 3.

I am sure they are working hard on wrapping a stable version of the admin panel including docs :) There are a milestone for it here: https://github.com/nodes-vapor/admin-panel/milestone/1

**Comment by steffendsommer** [comment]
Hi @dekhaus 

Thanks for reaching out. Once our Vapor 3 version is ready, we'll make sure to have documentation for this as well. Docs are tracked here https://github.com/nodes-vapor/admin-panel/issues/25 and that issue is part of the Vapor 3 milestone. We hope to have this ready within a couple of weeks.

I hope that @joscdk's comment was helpful in terms of getting up and running in the meantime.

I'm closing this as we're tracking the docs elsewhere, but feel free to open another issue if you have any questions/issues.
